### PR TITLE
{chem}[GCC/11.3.0] GOMC v2.76-20230613 w/ CUDA 11.7.0

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/g/GOMC/GOMC-2.76-20230613-GCC-11.3.0-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+install_cmd = 'cp'
+installopts = '%(builddir)s/easybuild_obj/%(name)s_{CPU,GPU}_* %(installdir)s/bin'
+
+_gomc_exe = ['GOMC_CPU_GCMC', 'GOMC_CPU_GEMC', 'GOMC_CPU_NPT', 'GOMC_CPU_NVT', 'GOMC_GPU_GCMC',
+             'GOMC_GPU_GEMC', 'GOMC_GPU_NPT', 'GOMC_GPU_NVT']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in _gomc_exe],
+    'dirs': [],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/g/GOMC/GOMC-2.76-20230613-GCC-11.3.0-CUDA-11.7.0.eb:45]:
```suggestion
}

sanity_check_commands = ['%s | grep "Info: GOMC"' % x for x in _gomc_exe]
```

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**